### DISCUSSION TITLE
[bug] Missing or invalid. request() method: eth_sendTransaction

### DISCUSSION THREAD

**Discussion by gabrielstoica** [reown-com/appkit#2396] [Q&A]
### Link to minimal reproducible example

https://github.com/WalletConnect/web-examples/blob/main/advanced/wallets/react-wallet-v2/src/hooks/useWalletConnectEventsManager.ts#L163

### Summary

When trying to send a transaction through a smart account (connected to it through a wallet connect URI) via `session_request` I'm getting the following error:

<img width="1817" alt="Screenshot 2024-05-30 at 22 08 19" src="https://github.com/WalletConnect/web3modal/assets/33158000/c5ae3fc1-a2ed-417a-b48d-4d98495976d9">

I'm using the exact same code provided in the `minimal reproducible example above`. The event is not even triggered at all.

This video describe best what I'm trying to achieve: https://x.com/stoicaxyz/status/1778710389091447227. This was working few days ago, I could connect to Uniswap with a smart contract through my EOA, then swap ETH for WETH. Now, I can still connect but when trying to send a transaction (i.e. swap) I get the error above.

Looking forward to understanding where's the problem! Thanks!

### List of related npm package versions

@web3modal/wagmi": "^4.1.5"
@walletconnect/web3wallet": "1.12.1"
@wagmi/core: "^2.6.16",

[a comment by an automated account is not shown]

**Comment by gabrielstoica** [comment]
@glitch-txs do you know what can be the problem? Thanks!

**Comment by glitch-txs** [comment]
Hi could you share a MRE? You can test the wallet from our examples here https://react-web3wallet.vercel.app/
